### PR TITLE
Write non-OK harness statuses to a log file

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
I've triggered https://github.com/web-platform-tests/results-analysis/actions/runs/4195848233 to test this. If that pushes a `non-ok-harness-statuses.txt` file to the `non-ok-harness-statuses-test` branch it worked.

**Comment by foolip** [comment]
The remaining step after this will be a setup similar to https://github.com/web-platform-tests/interop/pull/288 that copies this file into the interop repo, and sends PRs if it changes.

**Review by DanielRyanSmith — COMMENTED** [review]
The logic looks good to me barring some comments

File: .github/workflows/update_gh_pages.yml [hunk 1]
```diff
@@ -24,12 +25,12 @@ jobs:
...
-    - name: Build app
+    - name: Build
       run: ./build.sh
 
     - name: Deploy to gh-pages/
       uses: JamesIves/github-pages-deploy-action@releases/v3
       with:
         ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
-        BRANCH: gh-pages
+        BRANCH: non-ok-harness-statuses-test
```
[the remark this answers is not in the document]
Comment (foolip) [reply .github/workflows/update_gh_pages.yml:35]: Yes, I'll revert this!

**Comment by foolip** [comment]
Abandoning this, we could never agree on the best approach in the interop team, so the status quo stands.

**Pull request closed without merging**